labs/lab08-airgap-lite/README.md

```diff
@@ -0,0 +1,608 @@
+
+![Log In](img/admin-console-login.png)
+
+Until this point, this server is just running Docker, Kubernetes, and the kotsadm containers.
+The next step is to upload a license file so KOTS can validate which application is authorized to be deployed. Use the license file we downloaded earlier.
+
+Click the Upload button and select your `.yaml` file to continue, or drag and drop the license file from a file browser. 
+
+![Upload License](img/upload-license.png)
+
+After you upload your license, you'll be greeted with an Airgap Upload screen. Select **choose a bundle to upload** and use the "application bundle" that you
+downloaded to your workstation using the customer portal here. Click **Upload Air Gap bundle** to continue the upload process.
+
+![airgap-upload](img/airgap-upload.png)
+
+You'll see the bundle uploaded and images being pushed to kURL's internal registry. This will take a few minutes to complete.
+
+![airgap-push](img/airgap-push.png)
+
+Once uploaded `Preflight Checks` will run. These are designed to ensure this server has the minimum system and software requirements to run the application.
+Depending on your YAML in `preflight.yaml`, you may see some of the example preflight checks fail.
+If you have failing checks, you can click continue -- the UI will show a warning that will need to be dismissed before you can continue.
+
+![Preflight Checks](https://kots.io/images/guides/kots/preflight.png)
+
+
+We'll find that the application is unavailable. 
+
+![app-down](img/app-down.png)
+
+While we'll explore [support techniques for airgapped environments](#collecting-a-cli-support-bundle) 
+below, in this case you should observe that our deployment is simply not valid, specifically, the
+standard nginx entrypoint has been overriden:
+
+```yaml
+      containers:
+        - name: nginx
+          image: nginx
+          command:
+            - exit
+            - "1"
+```
+
+So we'll need to create a new release in order to fix this.
+
+***
+## Deploying a new version
+
+From the `labs/lab05-airgap` directory, update the `manifests/nginx-deployment.yaml` file to remove the command override as shown below.
+
+
+```diff
+diff --git a/labs/lab05-airgap/manifests/nginx-deployment.yaml b/labs/lab05-airgap/manifests/nginx-deployment.yaml
+index fa29e8d..3a66405 100644
+--- a/labs/lab05-airgap/manifests/nginx-deployment.yaml
++++ b/labs/lab05-airgap/manifests/nginx-deployment.yaml
+@@ -16,9 +16,6 @@ spec:
+       containers:
+         - name: nginx
+           image: nginx:latest
+-          command:
+-            - exit
+-            - "1"
+           volumeMounts:
+             - mountPath: /usr/share/nginx/html
+               name: html
+```
+
+Once you're satisfied with your `nginx-deployment.yaml` create a new release with `make release`.
+You'll need to ensure you have your `REPLICATED_APP` and `REPLICATED_API_TOKEN` set. See the **Getting Started** section for information on how to obtain and set these.
+
+```shell
+make release
+```
```

Comment:
Replace with:

We already created a new Release in Vendor Portal with the fixed container. In order to make the release available, go to `Releases > Sequence 2` and click `Promote`. Select the `lab08-airgap` channel to promote it to.

Reply:
@jdewinne wanna pull this in yourself on the next iteration?